Sources/SwiftGenKit/Parsers/Fonts/CTFont.swift

```diff
@@ -5,27 +5,87 @@
 //
 
 import AppKit.NSFont
+import CoreText
 import Foundation
 import PathKit
 
 extension CTFont {
-  static func parse(file: Path, relativeTo parent: Path? = nil) -> [Fonts.Font] {
+  static func parse(file: Path, relativeTo parent: Path? = nil, extractCodePoints: Bool = false) -> [Fonts.Font] {
     let descs = CTFontManagerCreateFontDescriptorsFromURL(file.url as CFURL) as NSArray?
     guard let descRefs = (descs as? [CTFontDescriptor]) else { return [] }
 
+    // swiftlint:disable:next closure_body_length
     return descRefs.compactMap { desc -> Fonts.Font? in
       let font = CTFontCreateWithFontDescriptorAndOptions(desc, 0.0, nil, [.preventAutoActivation])
       let postScriptName = CTFontCopyPostScriptName(font) as String
       guard let familyName = CTFontCopyAttribute(font, kCTFontFamilyNameAttribute) as? String,
         let style = CTFontCopyAttribute(font, kCTFontStyleNameAttribute) as? String else { return nil }
 
       let relPath = parent.flatMap { file.relative(to: $0) } ?? file
+
+      let characterSet = CTFontCopyCharacterSet(font) as CharacterSet
+      let cgFont = CTFontCopyGraphicsFont(font, nil)
+
+      var icons: [String: String] = [:]
+
+      if extractCodePoints {
+        for unicode in characterSet.codePoints {
+          guard let unicodeScalar = UnicodeScalar(unicode) else { continue }
+
+          let utf16codepoints = Array(Character(unicodeScalar).utf16)
+
+          var glyphs = [CGGlyph](repeating: 0, count: utf16codepoints.count)
+
+          // Gets the Glyph
+          guard CTFontGetGlyphsForCharacters(font, utf16codepoints, &glyphs, utf16codepoints.count) == true else {
+            print("Failed to get glyph for character", unicode, utf16codepoints, unicodeScalar)
```

Comment:
We have a `logMessage` function to generate warnings and errors during parsing, you should use that instead of `print` here